.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,347 @@
+feedback along with the next steps needed to get your contribution merged with
+master.  In certain cases the code reviewer(s) or interested committers may help
+you rework the code, but generally you will simply be given feedback for you to
+make the necessary changes.
+
+This process will continue until the code is finally accepted.
+
+<a name="CodeAcceptance" />
+
+### 5.3 Acceptance
+
+Once your code is accepted, it will be integrated with the master branch.
+Typically it will be rebased and fast-forward merged to master as we prefer to
+keep a clean commit history over a tangled weave of merge commits.  However,
+regardless of the specific merge method used, the code will be integrated with
+the master branch and the pull request will be closed.
+
+Rejoice as you will now be listed as a
+[contributor](https://github.com/bcoin-org/bcoin/graphs/contributors)!
+
+<a name="FrequentlyAskedQuestions" />
+
+### 6. Frequently Asked Questions
+
+<a name="Balances" />
+
+### 6.1. Balances
+
+bcoin handles balances as follows:
+
+confirmed: balance computed only using confirmed transactions.
+
+unconfirmed: balance computed using both confirmed transactions and transactions pending in the mempool.
+
+This is done to properly handle change outputs of large withdrawals.
+
+<a name="Confirmations" />
+
+### 6.2. Confirmations
+
+bcoin wallet transactions are considered confirmed after 1 block. If your
+application needs to wait for more confirmations, you can wait for the
+corresponding number of blocks to be mined.
+
+<a name="Standards" />
+
+### 7. Contribution Standards
+
+<a name="Checklist" />
+
+### 7.1. Contribution Checklist
+
+- All changes are node >= 8.0.0 compliant
+- [&nbsp;&nbsp;] The code being submitted is commented according to the
+  [Code Documentation and Commenting](#CodeDocumentation) section
+- [&nbsp;&nbsp;] For new code: Code is accompanied by tests which exercise both
+  the positive and negative (error paths) conditions (if applicable)
+- [&nbsp;&nbsp;] For bug fixes: Code is accompanied by new tests which trigger
+  the bug being fixed to prevent regressions
+- [&nbsp;&nbsp;] Changes to modules, classes or functions are documented
+  properly using jsdoc, running `npm run docs` does not report any issues
+- [&nbsp;&nbsp;] Any new logging statements use an appropriate subsystem and
+  logging level
+- [&nbsp;&nbsp;] Running `npm run lint` does not report any issues
+- [&nbsp;&nbsp;] Running `npm run test` does not fail any tests
+
+<a name="Licensing" />
+
+### 7.2. Licensing of Contributions
+
+All contributions must be licensed with the
+[MIT license](https://github.com/bcoin-org/bcoin/blob/master/LICENSE).  This is
+the same license as all of the code in bcoin.
+
+This document is based on contribution guidelines for
+[btcd](https://github.com/btcsuite/btcd/), distributed under the [ISC
+LICENSE](https://github.com/btcsuite/btcd/blob/master/LICENSE).
+
+TODO: Responsible Disclosure
```

Comment:
I think we should make the RD document a separate file. What do you think?

Reply:
Agree with this, take it out for now so we can merge :-)